File: leetCode/jio/912. Sort an Array_bubbleSort.java
class Solution {
    public int[] sortArray(int[] nums) {
        
        int size = nums.length;
        
        for(int i = 0; i < size-1; i++){
               
            for(int j = 0; j < size-1-i; j++) {
                
                if(nums[j] > nums[j+1]){
                
                    int temp = nums[j+1];
                    nums[j+1] = nums[j];
                    nums[j] = temp;
                }
            }           
        }
        return nums;
    }
}

File: leetCode/heeseok/912. Sort an Array_Bubble_sort.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        numsSize = len(nums)
        for i in range(0, numsSize):
            for j in range(0, numsSize-1):
                if nums[j] > nums[j + 1]:
                    tmp = nums[j]
                    nums[j] = nums[j+1]
                    nums[j+1] = tmp

        return nums

